roles/nethermind/defaults/main.yaml

```diff
@@ -0,0 +1,44 @@
+nethermind_user: nethermind
+nethermind_datadir: /data/nethermind
+nethermind_auth_jwt_path: /data/execution-auth.secret
+
+nethermind_cleanup: false # when set to "true" it will remove the container
+
+nethermind_ports_p2p: 30303
+nethermind_ports_http_rpc: 8545
+nethermind_ports_engine: 8551
+nethermind_ports_metrics: 6060
+
+nethermind_container_name: nethermind
+nethermind_container_image: nethermind/nethermind:latest
+nethermind_container_env: {}
+nethermind_container_ports:
+  - "127.0.0.1:{{ nethermind_ports_http_rpc }}:{{ nethermind_ports_http_rpc }}"
+  - "127.0.0.1:{{ nethermind_ports_engine }}:{{ nethermind_ports_engine }}"
+  - "{{ nethermind_ports_p2p }}:{{ nethermind_ports_p2p }}"
+  - "{{ nethermind_ports_p2p }}:{{ nethermind_ports_p2p }}/udp"
+nethermind_container_volumes:
+  - "{{ nethermind_datadir }}:/data"
+  - "{{ nethermind_auth_jwt_path }}:/execution-auth.jwt:ro"
+nethermind_container_stop_timeout: "300"
+nethermind_container_networks: []
+nethermind_container_entrypoint:
+  - /nethermind/Nethermind.Runner
+nethermind_container_command:
+  - --datadir=/data
+  - --KeyStore.KeyStoreDirectory=/data/keystore
+  - --Network.ExternalIp={{ ansible_host }}
+  - --Network.P2PPort={{ nethermind_ports_p2p }}
+  - --Network.DiscoveryPort={{ nethermind_ports_p2p }}
+  - --JsonRpc.Enabled=true
+  - --JsonRpc.Host=0.0.0.0
+  - --JsonRpc.Port={{ nethermind_ports_http_rpc }}
+  - --Init.WebSocketsEnabled=true
+  - --JsonRpc.WebSocketsPort={{ nethermind_ports_http_rpc }}
+  - --JsonRpc.JwtSecretFile=/execution-auth.jwt
+  - --JsonRpc.EngineHost=0.0.0.0
+  - --JsonRpc.EnginePort={{ nethermind_ports_engine }}
+  - --Metrics.Enabled=true
+  - --Metrics.NodeName={{ inventory_hostname_short }}
+  - --Metrics.ExposePort={{ nethermind_ports_metrics }}
+nethermind_container_command_extra_args: []
```

Comment:
Would the idea be that bootnode definitions happen via `extra_args`? 
We've done it like this so far:
```
  {% if (bootnode_enrs is defined) and bootnode_enrs %}
  --boot-nodes="{{ bootnode_enrs | join(',') }}"
  {% endif %}
```

I guess it should make no difference to do it via extra args since it won't conflict with anything else.

Reply:
Yes. I tried to keep the default commands to something minimal. Anything extra, e.g. for our testnets, can be injected via the extra_args vars.